lib/code_corps/emails/receipt_email.ex

```diff
@@ -0,0 +1,46 @@
+defmodule CodeCorps.Emails.ReceiptEmail do
+  import Bamboo.Email
+  import Bamboo.PostmarkHelper
+
+  alias CodeCorps.Emails.BaseEmail
+  alias CodeCorps.{Repo, StripeConnectCharge, StripeInvoice}
+
+  def create(%StripeConnectCharge{} = charge) do
+    charge = preload(charge)
+
+    BaseEmail.create
+    |> to(charge.user.email)
+    # TODO: Put proper template id here, probably from env
+    |> template("id_of_template", charge |> build_model)
+  end
+
+  defp preload(%StripeConnectCharge{} = charge) do
+    # TODO: If there were a relationship to invoice, we could do the full necessary preload here
+    Repo.preload(charge, [:user, [stripe_connect_account: :organization]])
+  end
+
+  defp build_model(%StripeConnectCharge{} = charge) do
+    %{
+      user_first_name: charge.user.first_name,
+      project_title: project_title(charge),
+      charge_amount: charge.amount,
+      charge_statement_descriptor: charge.statement_descriptor,
+      high_five_image_url: high_five_image_url
+    }
+  end
+
+  # TODO: Maybe add invoice relationship to charge and let it
+  # load from there?
+  defp project_title(%StripeConnectCharge{invoice_id_from_stripe: invoice_id_from_stripe}) do
+    invoice =
+      StripeInvoice
+      |> Repo.get_by(id_from_stripe: invoice_id_from_stripe)
+      |> Repo.preload(stripe_connect_subscription: [stripe_connect_plan: :project])
```

Comment:
It's entirely possible that the invoice event has not yet hit our database. And this only works for subscriptions, since there is no invoice for one-off charges.

Perhaps we should be making use of `metadata` in order to associate these. I see no way we could otherwise associate a charge (to an organization) with a project (where the donation is actually made).